Moritz's patch to disable xmlto validation.  It fails with the latest
xmlto and this patch fixes #491820

diff --git a/doc/Makefile b/doc/Makefile
index 2ba565f..30cf775 100644
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -150,7 +150,7 @@ option-%.frag.xhtml: $(ELINKS) $(HELP2XML)
             cmd_jw = $(JW) -b $(2) $<
 
    quiet_cmd_xmlto = '    [$(LINK_COLOR)XMLTO$(END_COLOR)]   $(RELPATH)$@'
-         cmd_xmlto = $(XMLTO) -o $(call outdir) $(3) $(2) $<
+         cmd_xmlto = $(XMLTO) --skip-validation -o $(call outdir) $(3) $(2) $<
 
 quiet_cmd_pod2html = ' [$(LINK_COLOR)POD2HTML$(END_COLOR)]   $(RELPATH)$@'
       cmd_pod2html = $(POD2HTML) --outfile=$@ < $<
